src/users/users.controller.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  BadRequestException,
+  Body,
+  Controller,
+  Get,
+  HttpException,
+  HttpStatus,
+  Param,
+  Post,
+} from '@nestjs/common';
+import { CreateUserDto, LoginUserDto } from '../dto/users-dto';
+import { UsersService } from './users.service';
+
+@Controller('users')
+export class UsersController {
+  constructor(private readonly usersService: UsersService) {}
+
+  @Post('/signup')
+  createUser(@Body() createUserDto: CreateUserDto): object {
+    const data = this.usersService.createUser(createUserDto);
+    return data;
+  }
+
+  @Post('/signin')
+  loginUser(@Body() loginUserDto: LoginUserDto): object {
+    console.log(loginUserDto);
+    const data = this.usersService.loginUser(loginUserDto);
+    return data;
+  }
+
+  @Get('/:id')
+  getUserById(@Param('id') id: string): object {
```

Comment:
1. Can you require the input id as the `integer` instead of `string`?
2. What is the format of the `id`? Is it pure integer or uuid? It is better to use the later.

Reply:
modify to be UUID to user_id